CMakeLists.txt

```diff
@@ -41,8 +46,11 @@ set(TTMLIR_BINARY_DIR ${PROJECT_BINARY_DIR})
 set(LLVM_LIT_TOOLS_DIR "${TTMLIR_TOOLCHAIN_DIR}/src/llvm-project/llvm/utils/lit")
 include_directories(SYSTEM ${LLVM_INCLUDE_DIRS})
 include_directories(SYSTEM ${MLIR_INCLUDE_DIRS})
+include_directories(${TTMLIR_TOOLCHAIN_DIR}/src/stablehlo)
+include_directories(${TTMLIR_TOOLCHAIN_DIR}/src/stablehlo-build)
```

Comment:
If it was installed into the env, shouldn't the headers and libs be in the usual include and lib dirs?  Why do we need to include the source path?

Reply:
Yes that's the case normally, but for stablehlo they don't have an "include" directory and this is the path to their project base.